syntax = "proto3";

package rest_grpc_dual_mode;

service GreetingsGenerator {
    rpc GRPCGetGreetings (RequestMessage) returns (ResponseMessage) {}
}

message RequestMessage {
    string dataCommType = 1;
    string messageType = 2;
    int32 year = 3;
}

message ResponseMessage {
    string greetings = 1;
    bool messageReceived = 2;
}